floodviz/map_utils.py

```diff
@@ -20,8 +38,19 @@ def site_dict(site_list, url_prefix):
     # get data from url
     req = requests.get(url)
```

Comment:
Still need to handle non 200 status code responses. In this case, I would return None and print out a message. Your function comment should reflect that if the service call fails that the function will return None.